docs/excel/excel-add-ins-ranges-advanced.md

```diff
@@ -350,8 +350,41 @@ Excel.run(function (context) {
 
 You can also find the cell responsible for spilling into a given cell by using the [Range.getSpillParent](/javascript/api/excel/excel.range#getspillparent--) method. Note that `getSpillParent` only works when the range object is a single cell. Calling `getSpillParent` on a range with multiple cells will result in an error being thrown (or a null range being returned for `Range.getSpillParentOrNullObject`).
 
+## Trace formula precedents
+
+An Excel formula often refers to other cells or groups of cells. These cells or groups of cells are known as "precedents" to the formula. To learn more about Excel features related to relationships between cells, see the [Display the relationships between formulas and cells](https://support.microsoft.com/office/display-the-relationships-between-formulas-and-cells-a59bef2b-3701-46bf-8ff1-d3518771d507) article. 
+
+With [Range.getDirectPrecedents](/javascript/api/excel/excel.range#getdirectprecedents--), your add-in can locate the direct precedents to a formula. The `getDirectPrecedents` method can trace precedents to a formula within a single worksheet and across worksheets within a single workbook. This method can't trace precedent cells across workbooks. 
+
+The following sample calls the `getDirectPrecedents` method on the active range, loads the addresses of the direct precedent cells, and then loops through each precedent cell and changes its background color to yellow. 
+
+> [!NOTE]
+> The active range must contain a formula that references other cells in the same workbook for this sample to work properly. 
+
+```js
+Excel.run(function (context) {
+    // Precedents are cells referenced by the formula in a cell.
+    var range = context.workbook.getActiveCell();
+    var directPrecedents = range.getDirectPrecedents();
+    range.load("address");
+    directPrecedents.areas.load("address");
+    return context.sync();
+
+    console.log(`Direct precedent cells of ${range.address}:`);
+
+    // Use the direct precedents API to loop through precedents of the active cell.
+    for (var i = 0; i < directPrecedents.areas.items.length; i++) {
+      // Highlight and print out the address of each precedent cell.
+      directPrecedents.areas.items[i].format.fill.color = "Yellow";
+      console.log(`  ${directPrecedents.areas.items[i].address}`);
+    }
+    return context.sync();
+}).catch(errorHandlerFunction);
+```
+
 ## See also
 
 - [Work with ranges using the Excel JavaScript API](excel-add-ins-ranges.md)
 - [Excel JavaScript object model in Office Add-ins](excel-add-ins-core-concepts.md)
 - [Work with multiple ranges simultaneously in Excel add-ins](excel-add-ins-multiple-ranges.md)
+- [Display the relationships between formulas and cells](https://support.microsoft.com/office/display-the-relationships-between-formulas-and-cells-a59bef2b-3701-46bf-8ff1-d3518771d507)
```

Comment:
We don't have the other SOC topics linked down here. I think the logic behind that it that if the want to learn about the specific feature correlated with an API, they can check out SOC, but the average developer trying to navigate our doc set won't want to.

Reply:
I see your point. The other Support docs referenced in this article are not linked in this See also section. I added the link here because a Support doc is linked in the Comments article See also section (I was working on both in tandem and cross-referenced for style and structure). If the general rule is that we don't reference Support docs in the See also section, should I remove the Support reference from the Comments article See also section?

Reply:
I think it's okay in the Comments article because it's targeted to a single feature. "Range" is a broad collection of Excel features, so links to SOC (support.office.com) wouldn't be clearly applicable.